package.json

```diff
@@ -20,5 +20,9 @@
     "gulp-postcss": "^4.0.3",
     "gulp-sass": "^1.3.3",
     "gulp-sourcemaps": "^1.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "gulp"
```

Comment:
@pascalduez There is value! No need for other people to install gulp globally to start the package, as long as gulp is listed as a dev dependency. :smile_cat: